tests/serde.rs

```diff
@@ -0,0 +1,23 @@
+use secp256k1::*;
+
+// Public key for secret key: "0101010101010101010101010101010101010101"
+const DEFAULT_PUBLIC_KEY: &str = "\"BBuExVZ7EmRAmV0+1aq6BWXXHhg0YEgZ/5wX9enV3QePcL6vj1iLVBUH/tamQsWrQt/fgSCn9jneUSLUemmo6NE=\"";
+
+fn default_public_key() -> PublicKey {
```

Comment:
Suggest naming this to something else. Confusing with `PublicKey::default()`

Reply:
Yeah, that was a strange name choice!